### DISCUSSION TITLE
Adopt the Contributor Covenant for community guidelines

### DISCUSSION THREAD

**Discussion by brylie** [DISTRHO/Cardinal#395] [Ideas]
As projects mature and are widely used, community interactions tend to increase. To help foster harmonious community interactions, it is often helpful to articulate expectations for friendly behavior with examples and counter-examples to make the expectations more concrete. To this end, [many open-source communities](https://www.contributor-covenant.org/adopters/) are adopting the [Contributor Covenant](https://www.contributor-covenant.org/).

Since the Cardinal project is becoming widely used, it seems likely to benefit from having clear guidelines for community-friendly behavior from all involved. 

Let's discuss the idea of providing clear community guidelines for Cardinal users, contributors, evangelists, and maintainers alike 🙂

**Comment by falkTX** [comment]
I am generally not opposed to such things, but Cardinal did not exactly grew.
It is getting a little bit popular, but contributions will always be very little. Modular environment is very hard for most folks, and thus its own niche.

So I do not expect big discussions or patches to ever appear to be quite honest. Also, I don't want to deal with Cardinal as this big project that needs serious maintenance, it is just a hobby fun thing. Making it more than that would make it unfun.

My general rules for project maintenance are just 2:
1. Be nice
2. See  rule 1

If we are nice, things tend to go smoothly.

**Reply by dromer** [reply]
What would be useful is set up issue templates: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

Because we regularly get either issues reported with a lot of missing information, or requests that have already been answered and/or should go elsewhere.

**Reply by falkTX** [reply]
can agree with that. I have seen we can even have combo box with options, making it super easy to enter details (for example, instead of writing OS, version etc, just have those as options)

**Reply by brylie** [reply]
Yeah, templates would help. For example, I should have opened my initial question in the support requests discussion but overlooked the discussions when opening the issue.
